x-pack/plugins/fleet/common/types/index.ts

```diff
@@ -44,6 +44,9 @@ export interface FleetConfigType {
     disableRegistryVersionCheck?: boolean;
     bundledPackageLocation?: string;
   };
+  internal?: {
+    disableILMPolicies: boolean;
```

Comment:
I am late to the party but shouldn't this be optional? i.e. `disableILMPolicies?: boolean;`
Otherwise I think we'll need to do migration if we add more `internal` fields

Reply:
We do specify `defaultValue: false` here - https://github.com/elastic/kibana/pull/154460/files#diff-efd7c14ce4511cfc69744ae1c4de932cbd4f615a99dff917d6ca4366b6cb283fR162-R168

Not sure if that corresponds with being able to unset this in `kibana.yml` or if has to be a `schema.maybe` as well. cc @nchaulet